app/js/globals.js

```diff
@@ -4,6 +4,7 @@ require.config({
   baseUrl: '/',
   paths: {
     'gaia-dialog': 'components/gaia-dialog/gaia-dialog',
-    'gaia-component': 'components/gaia-component/gaia-component'
+    'gaia-component': 'components/gaia-component/gaia-component',
+    'gaia-icons': 'components/gaia-icons/gaia-icons'
```

Comment:
We have to be really careful when adding new components. The Sharing app has gone through a lot of work to get all dependencies loaded correctly. I strongly suggest trying this patch on a user variant build with no settings/prefs related to debugging set.